scripts/upgrade

```diff
@@ -88,61 +41,46 @@ fi
 # Remove SSOwat extension if it exists
 ssowat_extension=$(ynh_app_setting_get --app=$app --key=ssowat_extension)
 if [ ! -z "$ssowat_extension" ]; then
-	ynh_exec_warn_less ynh_composer_exec --phpversion=$phpversion --workdir=$final_path --commands="remove tituspijean/flarum-ext-auth-ssowat"
+	ynh_exec_warn_less ynh_composer_exec --phpversion=$phpversion --workdir=$install_dir --commands="remove tituspijean/flarum-ext-auth-ssowat"
 	ynh_app_setting_delete --app=$app --key=ssowat_extension
 fi
 
-#=================================================
-# CREATE DEDICATED USER
-#=================================================
-ynh_script_progression --message="Making sure dedicated system user exists..." --weight=1
-
-# Create a dedicated user (if not existing)
-ynh_system_user_create --username=$app --home_dir=$final_path
-
 #=================================================
 # DOWNLOAD, CHECK AND UNPACK SOURCE
 #=================================================
 
-if [ "$upgrade_type" == "UPGRADE_APP" ]
+if [ "$upgrade_type" == "UPGRADE_APP" ] && [ ynh_compare_current_package_version --comparison lt --version 1.0~ynh1 ]
```

Comment:
Hmmm for some reason it's not happy with that --comparison thingy:

`25421 WARNING ./upgrade: line 52: [: syntax error: `--comparison' unexpected`

I think what you mean is : 

```suggestion
if [ "$upgrade_type" == "UPGRADE_APP" ] && ynh_compare_current_package_version --comparison lt --version 1.0~ynh1
```

because the `[ ]` syntax expects a test using the `test` syntax (I mean the `test` bash/sh command)